### ISSUE TITLE
Reduced CSF guanidinoacetic acid concentration

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11237]
**Preferred term label:**

**Synonyms**
Low CSF guanidinoacetic acid concentration
Reduced CSF GAA concentration
Low CSF GAA concentration

**Definition (free text, please give PubMed ID)**
PMID: 28971744 GAA (otherwise known as glycocyamine) is further converted to creatine in the second step of the process.
Hence, AGAT deficiency is characterized by **low brain levels of both GAA** and creatine, accompanied by mild to moderate intellectual disability, delayed speech development and motor skills, and myopathy.

PMID: 20301745 Biochemical findings : 
AGAT deficiency. **Low GAA levels** in urine, plasma, or **CSF** and low or low-normal creatine and creatinine levels in urine, plasma, or CSF

**Parent term (use hpo.jax.org/app)**
Abnormal CSF amino acid concentration HP:0500184

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
L-Arginine:glycine amidinotransferase deficiency (AGAT deficiency) ORPHA:35704

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
added as HP:0020005
